### DISCUSSION TITLE
Is it possible to use biome as a library instead of CLI?

### DISCUSSION THREAD

**Discussion by shinebayar-g** [biomejs/biome#4737] [Q&A] [answered]
I'd like to invoke biome through NodeJS if possible instead of CLI?

**Comment by ematipico** [comment, marked as answer]
https://www.npmjs.com/package/@biomejs/js-api

**Reply by shinebayar-g** [reply]
Nice! Thanks for the link. https://www.npmjs.com/package/@biomejs/wasm-nodejs is this package going to be a little bit smaller? 14.4MB would be rather heavy dependency 😅

**Reply by ematipico** [reply]
No. That's the whole Biome binary compiled to WASM, and it's not likely to shrink with time, the opposite.

You can still consider compiling to WASM parts of Biome, depending what you need
